pkgs/development/libraries/xgboost/default.nix

```diff
@@ -143,6 +143,7 @@ stdenv.mkDerivation rec {
       "Scalable, Portable and Distributed Gradient Boosting (GBDT, GBRT or GBM) Library";
     homepage = "https://github.com/dmlc/xgboost";
     license = licenses.asl20;
+    meta.mainProgram = "xgboost";
```

Comment:
```suggestion
    mainProgram = "xgboost";
```

Reply:
This is the cause of the ofborg eval error.

`mainProgram` is not _strictly required_, since it will use pname as a fallback. But this behaviour is deprecated and may get removed in the future.